README.md

```diff
@@ -1,36 +1,41 @@
-# Lingo - K8s LLM Proxy + Scaler
+# Lingo - The lightweight model proxy
```

Comment:
I do think we should add K8s in the title so it's clear that this only works on K8s and is very focused towards K8s.

What about one of these options?
- The lightweight ML Model proxy and autoscaler for K8s
- Lightweight K8s ML Proxy and Autoscaler
- K8s ML Proxy and Autoscaler

Reply:
What about leaving the title `# Lingo` and keeping the description in the description section of the repo? I feel like these are too long for an H1.